courses/machine_learning/deepdive2/structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>Nit. Appears that other Lab Tasks use ### header instead of ## header. Replace with ### here for consistency. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I had 1-7 as ### because they were underneath ## Create Keras model heading. Train and evaluate was outside of that "heading scope" therefore I went back to ## heading type for it. So I added ## Run and evaluate model  above it so that I could downgrade task 8 to ###. I think it looks better now, at least my systematic :)</p>

Reply:
ok. yeah fair enough. thanks.